Applications/opensense/opensense.cpp

```diff
@@ -111,13 +121,23 @@ int main(int argc, char **argv)
                     return 0;
                 }
                 else if ((option == "-Transform") || (option == "-T")) {
+                    if (argc < 3) {
+                        cout << "Marker file is needed for this option. Please fix and retry." << endl;
+                        PrintUsage(argv[0], cout);
+                        exit(-1);
+                    }                   
                     std::string markerFile{ argv[i + 1] };
                     TimeSeriesTable_<SimTK::Quaternion> quaternions =
                         OpenSenseUtilities::createOrientationsFileFromMarkers(markerFile);
                     cout << "Done." << endl;
                     return 0;
                 }
                 else if ((option == "-AddIMUs") || (option == "-A")) {
+                    if (argc < 4) {
+                        cout << "Both model file (.osim) and markers file are needed for this option. Please fix and retry." << endl;
```

Comment:
`Both a model (.osim) file and marker data (e.g. .trc) file are necessary to add IMU frames to the model based-on marker data.`